Simplenote/Classes/SPNoteEditorViewController.m

```diff
@@ -712,7 +711,7 @@ - (void)bounceMarkdownPreview
                     self.noteEditorTextView.hidden = NO;
                     [snapshot removeFromSuperview];
 
-                    self->bounceMarkdownPreviewOnActivityViewDismiss = NO;
+                    self->_bounceMarkdownPreviewOnActivityViewDismiss = NO;
```

Comment:
Since we're there, and you've already added a property, mind dropping the direct ivar usage? 😄